GamesApp/GamesApp/Scenes/Dice/DiceWorker.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  DiceWorker.swift
+//  GamesApp
+//
+//  Created by xyz mac on 29.05.2022.
+//
+
+import UIKit
+
+protocol DiceWorkerLogic {
+    func rollTheDice() -> (buttonTitle: String, number: Numbers, stats: [Double])
```

Comment:
Here's dirty code to return complicated touple from public protocol's function. 
U r to create separate structure and return it.
Also u may use this structure include data flow Response in DiceModels.